glide-core/redis-rs/redis/src/aio/connection_manager.rs

```diff
@@ -309,4 +309,10 @@ impl ConnectionLike for ConnectionManager {
         // always return false due to automatic reconnect
         false
     }
+
+    fn get_az(&self) -> Option<String> {
```

Comment:
Add function description and what does it always return `None`

Reply:
removed